appveyor.yml

```diff
@@ -1,37 +1,38 @@
 os: Visual Studio 2015
 
 environment:
+  NVS_VERSION: 1.4.0
   matrix:
-    - nodejs_version: 0.10
-    - nodejs_version: 4
-    - nodejs_version: 6
-
-platform:
-  - x64
-  - x86
+    - NODEJS_VERSION: node/0.10
+    - NODEJS_VERSION: node/4
+    - NODEJS_VERSION: node/6
+    - NODEJS_VERSION: chakracore/8.1.2
 
 shallow_clone: true
 
 install:
-  - ps: Install-Product node $env:nodejs_version $env:Platform
   - ps: Set-ExecutionPolicy Unrestricted -Scope CurrentUser -Force
+  - git clone --branch v%NVS_VERSION% --depth 1 https://github.com/jasongin/nvs %LOCALAPPDATA%\nvs
+  - set PATH=%LOCALAPPDATA%\nvs;%PATH%
   # make node@4.x x86 builds work
   # https://github.com/mapbox/node-pre-gyp/issues/209#issuecomment-217690537
+  - nvs add %NODEJS_VERSION%
+  - nvs use %NODEJS_VERSION%
   - npm config set -g cafile=package.json
   - npm config set -g strict-ssl=false
   # upgrade node-gyp to dodge 2013 compile issue present in the node gyp bundled with node v0.10
   # https://github.com/nodejs/node-gyp/issues/972#issuecomment-231055109
   - npm install node-gyp@3.x
-  - node --version
-  - npm --version
   - node -e "console.log(process.arch);"
   - IF /I "%PLATFORM%" == "x64" set PATH=C:\Python27-x64;%PATH%
   - IF /I "%PLATFORM%" == "x86" SET PATH=C:\python27;%PATH%
   - IF /I "%PLATFORM%" == "x64" CALL "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" amd64
   - IF /I "%PLATFORM%" == "x86" CALL "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" x86
   - npm install
-  - npm test
```

Comment:
just to verify the cause, could you undo this change? Bring back `npm test` under `install` step?